tensorrt_llm/grpc/smg/__init__.py

```
@@ -0,0 +1,95 @@
+__all__ = [
+    "PROTOS_AVAILABLE",
+    "REQUEST_MANAGER_AVAILABLE",
+    "SERVICER_AVAILABLE",
+    "trtllm_service_pb2",
+    "trtllm_service_pb2_grpc",
+    "GrpcRequestManager",
+    "TrtllmServiceServicer",
+    "create_sampling_params_from_proto",
+    "create_lora_request_from_proto",
+    "create_disaggregated_params_from_proto",
+]
```

Ruff (0.16.0)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)